pkg/config/admin_secrets_test.go

```diff
@@ -14,15 +15,17 @@ func TestAdminSecrets(t *testing.T) {
 
 	testFile, err := ioutil.ReadFile("admin_secrets_test.yaml")
 	assert.NoError(t, err)
+	secretsFromFile := config.AdminSecretsConfig{}
+	err = yaml.Unmarshal(testFile, &secretsFromFile)
```

Comment:
do you not want to assert that err is nil?

Reply:
well if something went wrong then `secretsFromFile` won't be correct so the assertion will fail, but there's no hard in checking that it's not Nil.